Makefile

```diff
@@ -95,9 +95,6 @@ help: ## Display this help.
 .PHONY: manifests
 manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
 	$(CONTROLLER_GEN) rbac:roleName=manager-role crd webhook paths="./..." output:crd:artifacts:config=config/crd/bases
-ifeq ($(CONTROL_PLANE), kcp)
```

Comment:
You should be able to remove `CONTROL_PLANE` everywhere in this file

Reply:
Thanks! Updated.